### PR TITLE
feat(cvi): apply new controller design

### PR DESCRIPTION
**Pull request by Isteb4k**
## Description
Apply new CVI controller design

! The VD controller is disabled

## Checklist
- [X] The code is covered by unit tests.
- [ ] e2e tests passed.
- [X] Documentation updated according to the changes.
- [X] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: images/virtualization-artifact/pkg/controller/cvmi_controller.go [hunk 1]
```diff
@@ -17,60 +17,10 @@ limitations under the License.
...
-	"github.com/deckhouse/virtualization-controller/pkg/dvcr"
-	"github.com/deckhouse/virtualization-controller/pkg/sdk/framework/two_phase_reconciler"
 )
 
 const (
-	cvmiControllerName = "cvi-controller"
-	cvmiShortName      = "cvi"
-
 	ImporterPodVerbose    = "3"
 	ImporterPodPullPolicy = string(corev1.PullIfNotPresent)
```
Comment (diafour) [inline images/virtualization-artifact/pkg/controller/cvmi_controller.go:25]: Do we need these constants and this file? These constants present in new controller: cvi_controller.go.

**Review by diafour — CHANGES_REQUESTED** [review]
Some comments

File: images/virtualization-artifact/pkg/controller/cvmi_controller.go (hunk 1 above)
Comment (Isteb4k) [reply images/virtualization-artifact/pkg/controller/cvmi_controller.go:25]: Used by VI controller, should be removed in vi pull request

**Review by yaroslavborbat — APPROVED** [review]

**Review by diafour — APPROVED** [review]
Let's go!

**Pull request merged**